js/main.js

```diff
@@ -103,7 +103,7 @@ requirejs.config({
 		"negative-controls": "components/negative-controls",
 		"d3": "d3.min",
 		"d3_tip": "d3.tip",
-		"jnj_chart": "jnj.chart",
+		"jnj_chart": "https://cdn.rawgit.com/odysseusinc/atlascharts/master/index",
```

Comment:
Thanks @pavgra .  The issue here then is where this common codebase lives.  It shouldn't be removed from the repository under the OHDSI organization.  If we are looking to create a shared repository of common code, you should speak with @pbr6cornell about setting up this shared repository, and then we can work together on moving the code into a form that is better suited to code reuse.  But it's not appropriate to have the code moved from the OHDSI governance to some other organization without first asking the code owners first.